distribution/docs/src/main/resources/_contents/_endpoints/content-endpoint-contents.adoc

```diff
@@ -0,0 +1,530 @@
+==== Content Endpoint
+
+The Content REST endpoint provides a CDR REST Retrieve v2.0-compliant DDF endpoint that allows clients to perform CRUD operations on the Content Repository using REST, a simple architectural style that performs communication using HTTP.
+
+The Content REST CRUD endpoint provides the capability to read, create, update, and delete content in the Content Repository, as well as create, update, and delete metacards in the catalog provider, i.e., the Metadata Catalog (MDC). Furthermore, this endpoint allows the client to perform the create/update/delete operations on just the Content Repository, just the MDC, or both in one operation.
+
+[NOTE]
+====
+The Content Framework is currently transactional for create operations only. Therefore, the client sends a create request to create content in the DDF Content Repository, processes the content to create a metacard, and ingests it into the MDC (i.e.,directive=STORE_AND_PROCESS), If a problem is encountered during the catalog ingest, the content is removed from the DDF Content Repository, analogous to a rollback. This is so that the DDF Content Repository and the MDC are kept in sync.
+====
+
+The Content Framework does not support rollback capability for update or delete operations that affect both the DDF Content Repository and the MDC.
+
+===== Installing Content Endpoint
+
+The Content Endpoint is installed by default with a standard installation in the ${ddf-catalog} application.
+
+===== Configuring Content Endpoint
+
+The location used for content storage can be configured.
+
+include::{adoc-include}/_tables/org.codice.ddf.catalog.content.impl.FileSystemStorageProvider-table-contents.adoc[]
+
+===== Using Content Endpoint
+
+.Content Endpoint URL
+----
+${secure_url}/services/content
+----
+
+The Content REST CRUD endpoint provides the capability to read, create, update, and delete content in the DDF Content Repository as well as create, update, and delete metacards in the catalog provider as follows.
+
+.Content Endpoint Operations
+[cols="1,1m,3a,3" options="header"]
+|===
+|Operation
+|HTTP Request
+|Details
+|Example URL
+
+|Create Content and Catalog Entry
+|HTTP POST
+|The multipart/form-data REST request that contains the binary data to be stored in the DDF Content Repository and to be parsed to create a metacard for ingest into the MDC. This binary data can be included in the request’s body or as a file attachment.
+
+An HTTP 201 CREATED status code is returned to the client with:
```

Comment:
Might be worth noting that the 201 is returned immediately, so in the case of larger multipart uploads, you will get the id back before you can actually query for that id. This is consistent with the RESTful pattern, but can still catch people by surprise.